src/main/java/nextstep/optional/Expression.java

```diff
@@ -14,12 +16,9 @@ private static boolean matchExpression(Expression e, String expression) {
     }
 
     static Expression of(String expression) {
-        for (Expression v : values()) {
-            if (matchExpression(v, expression)) {
-                return v;
-            }
-        }
-
-        throw new IllegalArgumentException(String.format("%s는 사칙연산에 해당하지 않는 표현식입니다.", expression));
+        return Arrays.stream(values())
+                .filter(operator -> matchExpression(operator, expression))
```

Comment:
```suggestion
                .filter(operator -> operator.matches(expression))
```
operator에게 메시지를 던지면 어떨까요?

Reply:
그게 조금 더 책임을 명확하게 할 수 있겠네요ㅎㅎ 다음 PR에 리팩토링 반영해서 올리겠습니다!